src/main/kotlin/apply/domain/mission/Mission.kt

```diff
@@ -40,6 +42,9 @@ class Mission(
     val isSubmitting: Boolean
         get() = status == MissionStatus.SUBMITTING
 
+    val isDescriptionViewable: Boolean
+        get() = !hidden && (status == MissionStatus.SUBMITTING || status == MissionStatus.UNSUBMITTABLE)
```

Comment:
c: 향후 과제 제출물 제출 가능 날짜는 별도로 관리할 예정입니다.

```suggestion
        get() = !hidden && status in listOf(MissionStatus.SUBMITTING, MissionStatus.UNSUBMITTABLE)
```